button-listen.c

```diff
@@ -11,37 +14,77 @@ static double timespec_diff(struct timespec const* start, struct timespec const*
     return s + (ns / 1e9);
 }
 
+static bool open_file(FILE **evf, const char *filename)
+{
+    FILE *file = NULL;
+    int fd = -1;
+    file = fopen(filename, "rb");
+    if(!file) {
```

Comment:
Consistent styling would be nice ;).

(Feel free to steal the input's clang-format, which I *think* I may have committed?).

Reply:
I didn't see a style config sorry. I've tried to tidy it up a bit.

Reply:
`input/.clang-format` ;). May or may not match the existing style, though, as it postdates this code, IIRC.

Reply:
Cool. I've now run clang-format over it too.